FV3/conf/modules.fv3.stellar

```diff
@@ -0,0 +1,29 @@
+# NOTE: the "module purge" and loading of the module command are
+# handled by the module-setup.sh (or .csh) script.
+##
+## load programming environment
+## this typically includes compiler, MPI and job scheduler
+##
+
+# Kai's modules (note these are commented out in favor of the more up-to-date versions):
+# module load intel/19.1
+# module load openmpi/intel-19.1/4.1.0
+# module load netcdf/intel-19.1/hdf5-1.10.6/4.7.4
+# module load hdf5/intel-19.1/1.10.6
+
+# Up-to-date equivalents of Kai's modules:
+module load intel/2021.1.2
+module load openmpi/intel-2021.1/4.1.2
+module load netcdf/intel-2021.1/hdf5-1.10.6/4.7.4
+module load hdf5/intel-2021.1/1.10.6
+
+# Kai's environment variables
```

Comment:
Since we typically do not set this variable explicitly in modules files, it would be helpful to add some context for why:
```suggestion
# Per Kai-Yuan Cheng, the netcdf module on Stellar sets the NETCDFDIR variable
# rather than the NETCDF_DIR variable, the variable used on existing supported
# platforms.  For compatibility we therefore add a NETCDF_DIR variable to the
# environment which takes the value of the NETCDFDIR variable.
```